pkg/logutil/logutil.go

```diff
@@ -30,3 +31,23 @@ func NewWriterToFile(maxSize int, dirPath, name string) (io.Writer, error) {
 	return lw, nil
 
 }
+
+// TeeOutput tees out output to a file path specified by env var `UROOT_LOG_PATH` up to a max limit. Can optionally create the directory structure for the filepath.
+func TeeOutput(writer io.Writer, maxSize int, initDir bool) (io.Writer, error) {
+	logPath := os.Getenv("UROOT_LOG_PATH")
+	if logPath != "" {
+		if initDir {
+			if err := os.MkdirAll(filepath.Dir(logPath), 0700); err != nil {
+				return nil, err
+			}
+		}
+		lw, err := NewWriterToFile(maxSize, logPath)
+		if err != nil {
+			os.Setenv("UROOT_LOG_PATH", "")
```

Comment:
any reason we need set the environment var to be empty ?

Reply:
I guess I assume that if logging fails for one, we should give up on logging for the rest. Maybe an unreasonable assumption, I'll remove it and add the logic outside of the function if needed.